src/main/java/com/multiintstore/indexing/IndexingService.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) Connexta
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3
+ * of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the
+ * GNU Lesser General Public License is distributed along with this
+ * program and can be found at http://www.gnu.org/licenses/lgpl.html.
+ */
+
+package com.multiintstore.indexing;
+
+public class IndexingService {
```

Comment:
❓should the Indexing service live outside the MiS?

Reply:
Yeah, I'll remove this and add an `IndexingController`